# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/quota/test/xpcshell/xpcshell.toml
# Commit: aaa63956f3f0
# Full Hash: aaa63956f3f022575b29059d5909f63079723d3a
# Author: Jan Varga <jvarga@mozilla.com>
# Date: 2024-10-10 09:22:33
# Regressor Bug: 1866402
# File Overlap Count: 1
# Description:
#   Bug 1866402 - Add nsIQuotaManagerService::PersistentOriginInitialized and nsIQuotaManagerService::TemporaryOriginInitialized methods; r=dom-storage-reviewers,jari
#   
#   This additional testing infrastructure is needed for verifying origin
#   initialization status.
#   
# ==============================================================================

diff -r f799f35b1c84 -r aaa63956f3f0 dom/quota/test/xpcshell/xpcshell.toml
--- a/dom/quota/test/xpcshell/xpcshell.toml	Wed Oct 09 22:16:50 2024 +0000
+++ b/dom/quota/test/xpcshell/xpcshell.toml	Wed Oct 09 22:16:50 2024 +0000
@@ -79,8 +79,12 @@
 
 ["test_initializePersistentClient.js"]
 
+["test_initializePersistentOrigin.js"]
+
 ["test_initializeTemporaryClient.js"]
 
+["test_initializeTemporaryOrigin.js"]
+
 ["test_listOrigins.js"]
 
 ["test_originEndsWithDot.js"]
